index.html

```diff
@@ -253,5 +253,81 @@ <h3 class="aboutheading">About me</h3>
       </ul>  
     </section>
 
+<!-- Contact form -->
+
+    <section class="text-container">
+      <h2 class="newprojects-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+      <div class="contact-form">
+        <form action="https://formspree.io/f/xknylobk" method="post">
+          <input type="text" name="name" class="input-name" placeholder="Full Name" maxlength="30" required>
+          <input type="email" name="email" class="input-email" placeholder="Email Address" required>
+          <textarea name="message" id="message" maxlength="500" class="input-text-area" placeholder="Enter text here"></textarea>
+          <div class="btn">
```

Comment:
- [ ] Please know `textarea` should be required as well as it is an input field as well. 😁

Reply:
I have implemented this